ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh

```diff
@@ -6,27 +6,29 @@ set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
-
-KUBEVIRT_BASE_DOMAIN="tenant1.origin-on-kubevirt.dev.openshift.com"
-KUBEVIRT_API_VIP=192.168.123.15
-KUBEVIRT_INGRESS_VIP=192.168.123.20
-KUBEVIRT_KUBECONFIG=${HOME}/.kube/config
+KUBEVIRT_BASE_DOMAIN="testci.tenant1.dev.openshift.com"
+KUBEVIRT_API_VIP=10.123.124.15
+KUBEVIRT_INGRESS_VIP=10.123.124.20
+KUBEVIRT_CIDR="10.123.124.0/24"
 KUBEVIRT_NAMESPACE=tenantcluster
-KUBEVIRT_NETWORK_NAME=tenantcluster
+KUBEVIRT_NETWORK_NAME=mynet
 KUBEVIRT_TENANT_STORAGE_CLASS_NAME=standard
-KUBEVIRT_VOLUME_ACCESS_MODE=ReadWriteMany
+KUBEVIRT_VOLUME_ACCESS_MODE=ReadWriteOnce
```

Comment:
Why? this way we won't support live migration

Reply:
GCP not supported MANY